specification/communication/data-plane/CallAutomation/preview/2023-10-03-preview/communicationservicescallautomation.json

```diff
@@ -1709,6 +1698,246 @@
         }
       }
     },
+    "/calling/callConnections/{callConnectionId}/recordings": {
```

Comment:
Have we considered just reusing the existing endpoint and adding callConnectionId as another subtype of callLocator?

Reply:
We are doing this method now